specs/_features/maxeb_increase/capella.py

```diff
@@ -1776,6 +1788,49 @@ def process_execution_layer_withdraw_request(
     ))
 
 
+def is_consolidated(validator: Validator) -> bool:
+   validator.consolidated_to != UNSET_CONSOLIDATED_TO
+
+
+def resolve_slashed_index(state: BeaconState, index: ValidatorIndex) -> ValidatorIndex:
+    if is_consolidated(state.validators[index]):
+        # Recursively resolve consolidated index
+        return resolve_slashed_index(state, state.validators[index].consolidated_to)
+    else:
+        return index
+
+
+def process_consolidation(state: BeaconState, consolidation: Consolidation) -> None:
+    target_validator = state[consolidation.target_index]
+    source_validator = state[consolidation.source_index]
+
+    assert target_validator.exit_epoch == FAR_FUTURE_EPOCH
+    assert source_validator.exit_epoch == FAR_FUTURE_EPOCH
+
+    # verify source withdrawal credentials, which have authority over validating keys
+    assert source_validator.withdrawal_credentials[:1] == ETH1_ADDRESS_WITHDRAWAL_PREFIX
```

Comment:
```suggestion
    assert has_eth1_withdrawal_credential(source_validator) or has_compounding_withdrawal_credential(source_validator)
```

I think consolidation should be allowed for both of these cred types